### PR TITLE
DOC-2354 TinyMCE 7.0.1 Release notes & community changelog

### PR DESCRIPTION
**Pull request by FarzadHayat**
Ticket: DOC-2354

Site: [7.0.1 Release Notes](http://docs-feature-7-doc-2354.staging.tiny.cloud/docs/tinymce/latest/7.0.1-release-notes/#overview)
Site: [7.0.1 Changelog](http://docs-feature-7-doc-2354.staging.tiny.cloud/docs/tinymce/latest/changelog/#7-0-1-2024-04-10)

Changes:
* Add 7.0.1 staging template to `staging/docs-7` for Community and GA Docs Release.

Pre-checks:
- [x] Branch prefixed with `feature/7/` or `hotfix/7/`
- [x] `modules/ROOT/nav.adoc` has been updated `(if applicable)`
- [x] Files has been included where required `(if applicable)`
- [x] Files added for `New product features`, and included a `release note` entry.
- ~[ ] Bump `api-version` to `7.0.1`~ this will be done as `hotfix` post release.
- [x] added `7.0.1` changelog to `changelog.adoc`
- ~[-] add `7.0.1` EOS date to `support.adoc`~ (not a major/minor version, so does not need EOS date)

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/7.0.1-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+:keywords: releasenotes, new, changes, bugfixes
+:page-toclevels: 1
+
+include::partial$misc/admon-releasenotes-for-stable.adoc[]
+
+
+[[overview]]
+== Overview
+
+{productname} {release-version} was released for {enterpriseversion} and {cloudname} on Wednesday, April 10^th^, 2024. These release notes provide an overview of the changes for {productname} {release-version}, including:
```
Comment (kemister85) [inline modules/ROOT/pages/7.0.1-release-notes.adoc:14]:
```suggestion
{productname} {release-version} was released for {enterpriseversion} and {cloudname} on Wednesday, April 10^th^, 2024.

These release notes provide an overview of the changes for {productname} {release-version}, including:
```

File: modules/ROOT/pages/7.0.1-release-notes.adoc [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+=== Enhanced Code Editor
+
+The {productname} {release-version} release includes an accompanying release of the **Enhanced Code Editor** premium plugin.
+
+This **Enhanced Code Editor** release includes the following fixes.
+
+==== Text inside code editor was not properly rendered when using a dark {productname} skin.
+// #TINY-10784
+
+In previous versions of the **Enhanced Code Editor**, the text color of the code editor was inherited from the {productname} skin. This caused the light theme of the code editor to display white text on a white background when using a dark skin, such as `oxide-dark`.
```
Comment (kemister85) [inline modules/ROOT/pages/7.0.1-release-notes.adoc:34]: Should we link to [customize-ui#skins](https://www.tiny.cloud/docs/tinymce/latest/customize-ui/#skins) for dark mode?

```suggestion
In previous versions of the **Enhanced Code Editor**, the text color of the code editor was inherited from the {productname} skin. This caused the light theme of the code editor to display white text on a white background when using a dark skin, such as `oxide-dark`.
```

File: modules/ROOT/pages/7.0.1-release-notes.adoc [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+== Removed
+
+{productname} {release-version} also includes the following removal<s>:
+
+=== <TINY-vwxyz 1 changelog entry>
+// #TINY-vwxyz1
+
+// CCFR here.
+
+
```
Comment (kemister85) [inline modules/ROOT/pages/7.0.1-release-notes.adoc]:
```suggestion
```

File: modules/ROOT/pages/7.0.1-release-notes.adoc [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+== Known issues
+
+This section describes issues that users of {productname} {release-version} may encounter and possible workarounds for these issues.
+
+There <is one | are <number> known issue<s> in {productname} {release-version}.
+
+=== <TINY-vwxyz 1 changelog entry>
+// #TINY-vwxyz1
+
+// CCFR here.
```
Comment (kemister85) [inline modules/ROOT/pages/7.0.1-release-notes.adoc]:
```suggestion
```

File: modules/ROOT/nav.adoc [hunk 5]
```diff
@@ -397,7 +397,17 @@
...
+*** TinyMCE 7.0.1
+**** xref:7.0.1-release-notes.adoc#overview[Overview]
+**** xref:7.0.1-release-notes.adoc#accompanying-premium-plugin-changes[Accompanying Premium Plugin changes]
+**** xref:7.0.1-release-notes.adoc#improvements[Improvements]
+**** xref:7.0.1-release-notes.adoc#changes[Changes]
+**** xref:7.0.1-release-notes.adoc#removed[Removed]
+**** xref:7.0.1-release-notes.adoc#bug-fixes[Bug fixes]
+**** xref:7.0.1-release-notes.adoc#security-fixes[Security fixes]
+**** xref:7.0.1-release-notes.adoc#deprecated[Deprecated]
+**** xref:7.0.1-release-notes.adoc#known-issues[Known issues]
```
Comment (kemister85) [inline modules/ROOT/nav.adoc]:
```suggestion
** xref:release-notes.adoc[Release notes for {productname} {productmajorversion}]
*** TinyMCE 7.0.1
**** xref:7.0.1-release-notes.adoc#overview[Overview]
**** xref:7.0.1-release-notes.adoc#accompanying-premium-plugin-changes[Accompanying Premium Plugin changes]
**** xref:7.0.1-release-notes.adoc#bug-fixes[Bug fixes]
```

File: modules/ROOT/pages/7.0.1-release-notes.adoc [hunk 6]
```diff
@@ -0,0 +1,250 @@
...
+{productname} {release-version} was released for {enterpriseversion} and {cloudname} on Wednesday, April 10^th^, 2024. These release notes provide an overview of the changes for {productname} {release-version}, including:
+
+* xref:accompanying-premium-plugin-changes[Accompanying Premium plugin changes]
+* xref:improvements[Improvements]
+* xref:changes[Changes]
+* xref:removed[Removed]
+* xref:bug-fixes[Bug fixes]
+* xref:security-fixes[Security fixes]
+* xref:deprecated[Deprecated]
+* xref:known-issues[Known issues]
```
Comment (kemister85) [inline modules/ROOT/pages/7.0.1-release-notes.adoc]:
```suggestion
* xref:accompanying-premium-plugin-changes[Accompanying Premium plugin changes]
* xref:improvements[Improvements]
* xref:bug-fixes[Bug fixes]
```

**Review by kemister85 — COMMENTED** [review]
basic template cleanup

**Review by kemister85 — APPROVED** [review]
LGTM, nice work @FarzadHayat

**Review by spocke — APPROVED** [review]

**Pull request merged**